src/commands/sort.json

```diff
@@ -50,10 +50,10 @@
                 }
             },
             {
+                "description": "For the optional STORE keyword. It is marked 'unknown' because the keyword can appear anywhere in the argument array",
```

Comment:
we should use these descriptions in a few other commands. not sure if we have to in this PR though.
few candidates i can think of
SET, BITFIELD, 
maybe some of the commands listed [here](https://github.com/redis/redis/issues/10040#issuecomment-1013920954), [here](https://github.com/redis/redis/issues/10040#issuecomment-1013954262) (both have a comments column).
or modified here: https://github.com/redis/redis/pull/10167
i.e. in all of these we had some special concerns about flags and we weren't sure what to do at first..

Reply:
i added some more, please LMK if you can think of others I missed